node/db/src/memory.rs

```diff
@@ -42,7 +41,7 @@ impl Default for MemoryDB {
 
 impl DatabaseService for MemoryDB {}
 
-impl Write for MemoryDB {
+impl Store for MemoryDB {
```

Comment:
Whats the reason for collapsing Write and Read into Store?

Reply:
Annoying to reference both, no benefit for them being separated, conflicting trait name with io read and write